src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/controller/dto/CourtCase.kt

```diff
@@ -20,6 +21,16 @@ data class CourtCase(
       courtCaseEntity.latestCourtAppearance?.let { CourtAppearance.from(it) },
       courtCaseEntity.appearances.filter { it.statusId == EntityStatus.ACTIVE }.map { CourtAppearance.from(it) },
       courtCaseEntity.legacyData,
```

Comment:
this function is only used by: getCourtCaseByUUID
searchCourtCases (not the paged version)

I dont think anything calld that search endpoint - the getCourtCaseByUUID is used by the ui when editing appearances